src/app/blogs/[id]/page.tsx

```
@@ -20,27 +23,21 @@ const BlogDetailPage = () => {
-        dangerouslySetInnerHTML={{ __html: blog.body }}
-      />
-    </div>
+    <>
+      <BlogTemp
+        id={blog.id}
+        title={blog.title}
+        img={blog.topImage.url}
+        createdAt={blog.createdAt}
+        updatedAt={blog.updatedAt}
+      >
+        <div dangerouslySetInnerHTML={{ __html: blog.body }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)